x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -683,48 +683,17 @@ export function registerEncryptedSavedObjects(
       { key: 'ssl', dangerouslyExposeValue: true },
       { key: 'password', dangerouslyExposeValue: true },
     ]),
-    attributesToExcludeFromAAD: new Set([
-      'output_id',
-      'name',
-      'type',
-      'is_default',
-      'is_default_monitoring',
-      'hosts',
-      'ca_sha256',
-      'ca_trusted_fingerprint',
-      'config',
-      'config_yaml',
-      'is_preconfigured',
-      'proxy_id',
-      'version',
-      'key',
-      'compression',
-      'compression_level',
-      'client_id',
-      'auth_type',
-      'connection_type',
-      'username',
-      'sasl',
-      'partition',
-      'random',
-      'round_robin',
-      'hash',
-      'topics',
-      'headers',
-      'timeout',
-      'broker_timeout',
-      'required_acks',
-      'preset',
-      'secrets',
-    ]),
+    attributesToIncludeInAAD: new Set(['ssl', 'shipper', 'allow_edit']),
```

Comment:
do we have a way to change that list in the future we should probably remove `allow_edit` and `shipper` here?

Reply:
Unfortunately, we don't have a way that is compatible with zero-downtime upgrades, to remove fields that are already included in AAD. This may change in the future if we implement an versioned encrypted saved object definition storage mechanism.